styles.css

```diff
@@ -0,0 +1,949 @@
+  width: 100%;
+}
+
+.navbar {
+  max-width: 1440px;
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: 100%;
+  max-height: 100%;
+  padding: 0px 20px;
+}
+
+.navbar > ul {
+  align-items: center;
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  height: 100%;
+}
+
+.navbar > ul > li > a {
+  color: inherit;
+  font-weight: bold;
+  font-size: 32px;
+  text-decoration: inherit;
+  padding: 0px 20px;
+  display: block;
+  height: 100%;
+  animation: itemsHoverOut 0.3s both;
+  transition: color 0.2s ease;
+}
+
+.navbar > ul > li:hover > a {
+  color: var(--color-orange-dark);
+  animation: itemsHoverIn 0.3s both;
+}
+
+.burger {
+  display: none;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 30px;
+  height: 25px;
+  cursor: pointer;
+}
+
+.burger span {
+  display: block;
+  height: 4px;
+  background-color: black;
+  border-radius: 4px;
+  transition: transform 0.3s, opacity 0.3s;
+}
+
+.nav-menu {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  transition: transform 0.3s ease;
+}
+
+.nav-menu.active {
+  transform: translateX(0);
+  height: 100vh;
+}
+
+@media (max-width: 1000px) {
+  .burger {
+    display: flex;
+  }
+
+  .nav-menu {
+    position: absolute;
+    top: var(--header-height);
+    left: 0;
+    flex-direction: column;
+    width: 100%;
+    background-color: var(--color-light-white);
+    transform: translateX(100%);
+    transition: transform 0.3s ease;
+    height: 100vh;
+  }
+
+  .nav-menu > li > a {
+    padding: 15px;
+    font-size: 20px;
+    text-align: center;
+    width: 100%;
+  }
+}
+
+main {
+  position: relative;
+  max-width: 1440px;
+  width: 100%;
+  background-color: var(--color-light-white);
+  display: flex;
+  flex-direction: column;
+}
+
+section {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  gap: 0px;
+  min-height: 800px;
+  width: 100%;
+  padding: 0px;
+  padding: 80px var(--padding-x);
+  box-sizing: border-box;
+}
+
+@media (max-width: 1200px) {
+  section {
+    min-height: 700px;
+  }
+}
+
+@media (max-width: 1000px) {
+  section {
+    min-height: 600px;
+  }
+}
+
+@media (max-width: 800px) {
+  section {
+    min-height: 550px;
+  }
+}
+
+@media (max-width: 600px) {
+  section {
+    min-height: 400px;
+  }
+}
+
+.footer {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (max-width: 600px) {
+  .footer {
+    font-size: 12px;
+  }
+}
+
+@keyframes itemsHoverIn {
+  0% {
+    transform: translateY(0);
+  }
+  100% {
+    transform: translateY(-10px);
+  }
+}
+
+@keyframes itemsHoverOut {
+  0% {
+    transform: translateY(-10px);
+  }
+  100% {
+    transform: translateY(0);
+  }
+}
+
+/* -------------------- sec1 ------------------------- */
+
+.sec1 {
+  position: relative;
+  display: flex;
+  flex-direction: row;
```

Comment:
no need to add 

flex-direction: row;
 justify-content: flex-start;
  align-items: flex-start;
its the default.

Reply:
there is default styling to the section element added

```
section {
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  min-height: 800px;
  width: 100%;
  padding: 0px;
  padding: 80px var(--padding-x);
  box-sizing: border-box;
}
```